src/util.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Get the current totals.
+ *
+ * @param {Object} cart The cart object.
+ * @returns {Number} The totals of the cart object, if available, else the totals of the cart progress bar preview.
+ */
+export function getCurrentTotal( cart ) {
+	if (
+		undefined !== cart &&
+		undefined !== cart.cartTotals.total_price &&
+		undefined !== cart.cartTotals.currency_minor_unit
```

Comment:
When cart is defined, those are always defined, they're the in the default response.

Reply:
Thanks for pointing this out. I've remove these duplicated checks.